LinkDotNet.ValidationExtensions.Tests/Dynamic/DynamicRangeTests.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using FluentAssertions;
+using Xunit;
+
+namespace LinkDotNet.ValidationExtensions.Tests;
+
+public class DynamicRangeTests
+{
+    [Fact(DisplayName = "Should not be valid if 'MinimumWeight' is grather than 'MaximumWeight'")]
+    public static void ShouldNotBeValidIfMinimumIsGratherThanMaximum()
```

Comment:
Small type
Greater instead of Grather

Reply:
wow, thank you
It's done :upside_down_face: